web/src/components/organisms/Common/CommentsPanel/hooks.ts

```diff
@@ -12,7 +12,7 @@ export default ({ threadId }: Params) => {
   const t = useT();
 
   const [createComment] = useAddCommentMutation({
-    refetchQueries: ["GetAsset", "GetItems"],
+    refetchQueries: ["GetAsset", "GetAssets", "GetItems"],
```

Comment:
Do you need both GetAsset and GetAssets, or can you drop the GetAsset now?

Reply:
Yes, we need it.
It will refetch only queries available on the current screen, so it will not run the 3 together (depending on the screen).